### PR TITLE
Add docs and color update images

### LINKED ISSUE #3084: [Enhance] Color Docs
**User Story:**
As Spark we want to enhance our color documentation since we updated to a new accessibly green.

**Notes:**
SME: Brielle

**AC:**
Add the following content to https://sparkdesignsystem.com/using-spark/foundations/color

Replace: Having a consistent color palette is one of the foundations of a unified user experience. These are the colors that have been chosen to represent the brand.
 
Replace With: Having a consistent color palette is one of the foundations of unified user experience. Our objective is to achieve human-centered design in everything we do. We take accessibility seriously and comply with WCAG 2.0 and contrast guidelines, and we do our best to account for the needs of all humans.

### PR DESCRIPTION
**Pull request by RVMendoza**
## What does this PR do?
- update documentation
- update images
- update stepper pearl color and create new variables for pearl on dark bg

Screen Shot 2020-04-23 at 3 52 14 PM
Screen Shot 2020-04-23 at 3 51 11 PM

### Associated Issue
Fixes #3084 #3081 

### Documentation
 - [x] Update Spark Docs

### Accessibility
- [x] New changes abide by [accessibility requirements](https://sparkdesignsystem.com/docs/accessibility)

### Browser Testing (current version and 1 prior)
  - [x] Google Chrome
  - [x] Google Chrome (Mobile)
  - [x] Mozilla Firefox
  - [x] Mozilla Firefox (Mobile)
  - [x] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [x] Microsoft Edge
  - [x] Apple Safari
  - [x] Apple Safari (Mobile)

### Design Review
 - [x] Design reviewed and approved

### CODE REVIEW

File: src/pages/using-spark/foundations/color.mdx [hunk 1]
```diff
@@ -24,8 +24,11 @@ export const colorsLightSrcSet = `${colorsOnLight} 560w, ${colorsOnLight2x} 1120
 export const colorsDarkSrcSet = `${colorsOnDark} 560w, ${colorsOnDark2x} 1120w, ${colorsOnDark3x} 1680w`;
 
 # Color
-Having a consistent color palette is one of the foundations of a unified user experience.
-These are the colors that have been chosen to represent the brand.
+Our objective is to achieve human-centered design in everything we do.
+
+Having a consistent color palette is one of the foundations of unified user experience.
+
+We take accessibility seriously and comply with WCAG 2.0 and contrast guidelines, and we do our best to account for the needs of all humans.
```
Comment (bobwise) [inline src/pages/using-spark/foundations/color.mdx:31]: 👏👏👏

File: html/settings/_settings.scss [hunk 2]
```diff
@@ -1782,6 +1785,11 @@ $sprk-stepper-icon-box-shadow-selected-spread: 8px !default;
 $sprk-stepper-icon-box-shadow-selected: 0 0 0
   $sprk-stepper-icon-box-shadow-selected-spread
   $sprk-stepper-icon-border-color-selected !default;
+/// The box shadow of the Stepper step icon on a dark background.
```
Comment (bobwise) [inline html/settings/_settings.scss:1788]:
```suggestion
/// The box shadow of the Stepper step icon on a dark background
```

File: src/pages/using-spark/foundations/color.mdx (hunk 1 above)
Comment (afebbraro) [inline src/pages/using-spark/foundations/color.mdx:31]: I thought I saw a comment from @yodasw16  that said to change 'humans' to people. Did you decide not too?

Comment (RVMendoza) [reply src/pages/using-spark/foundations/color.mdx:31]: oh, I didn't actually see that comment, but people does sound better.  I was basing it off of the issue. I'll make the change!

**Pull request merged**